docs/Attributes.md

```diff
@@ -1,51 +1,91 @@
 # Attributes
 
-Attributes are properties that can be assigned to an entire track or a single detection within a track.  Attributes are created by users utilizing the Settings panel in the main file browser location.  New attributes can be added or existing ones can be modified.
+Attributes are properties that can be assigned to an entire track or a single detection within a track. Attributes are created within a dataset using the TrackDetails panel.
 
 !!! info
-    Attributes are global on the server currently and can be seen or modified by anyone.  There are future plans to move attributes to be specific to projects or users.
+    Attributes were previously global but have been moved to being dataset specific.
```

Comment:
```suggestion
    Attributes were previously globally defined.  They are now defined on each individual dataset.
```

Reply:
Added clarity about definitions being the thing that changed scope.  Seems obvious, but I think it's useful to use the key terms.